src/overlays/actors/ovl_En_Zo/z_en_zo.c

```diff
@@ -78,41 +92,272 @@ static DamageTable D_8099F4E8 = {
     /* UNK_DMG_0x1E   */ DMG_ENTRY(0, 0x0),
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
+static ActorAnimationEntryS sAnimations[7] = {
+    { &D_06004248, 1.0f, 0, -1, 0, 0 },  { &D_06004248, 1.0f, 0, -1, 0, -4 }, { &D_06003610, 1.0f, 0, -1, 0, -4 },
+    { &D_06000598, 1.0f, 0, -1, 0, -4 }, { &D_06000D48, 1.0f, 0, -1, 0, -4 }, { &D_0600219C, 1.0f, 0, -1, 0, -4 },
+    { &D_06002898, 1.0f, 0, -1, 0, -4 },
+};
+static s8 D_8099F578[20] = { -1, 1, 12, 13, 14, 9, 10, 11, 0, 6, 7, 8, 3, 4, 5, 2, -1, -1, -1, -1 };
+static s8 D_8099F58C[16] = { 0, 0, 0, 0, 3, 4, 0, 6, 7, 0, 9, 10, 0, 12, 13, 0 };
+static u8 D_8099F59C[16] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+static u16 D_8099F5AC[16] = { 4000, 4, 1, 3, 6000, 4, 1, 6, 4000, 4, 1, 3, 6000, 4, 1, 6 };
+static Vec3f D_8099F5CC = { 400.0f, 0.0f, 0.0f };
+static Vec3f D_8099F5D8 = { 0.0f, 0.0f, 0.0f };
+static Gfx D_8099F5E8[3] = {
```

Comment:
The best I can do is put it all (except the `Vec3f`s and `Gfx`, which can go further) right above `LookAtPlayer`, since it uses `D_8099F5AC` which is later than the other data, but `LookAtPlayer` is above the functions where the other ones are used.

Reply:
Oh yeah, there's some in-function stuff lousing up the works? Hmm.

`D_8099F5AC` looks like it's in-function static.